.upstream-tests/utils/nvidia/linux/perform_tests.bash

```diff
@@ -237,7 +239,7 @@ then
   ARCH_DETECTION_LOG=$(mktemp)
 
   LIBCUDACXX_SITE_CONFIG=${LIBCUDACXX_LIT_SITE_CONFIG} \
-  bash -c "lit -vv -a ${LIBCUDACXX_PATH}/test/nothing_to_do.pass.cpp" \
+  bash -c "lit -vv -a ${LIBCUDACXX_PATH}/test/nothing_to_do.pass.cpp -Dcompute_archs=${KNOWN_COMPUTE_ARCHS}" \
```

Comment:
This expands into `-Dcompute_archs=35 50 52 53...`, no? Shouldn't it be `-Dcompute_archs='${KNOWN_COMPUTE_ARCHS}'`?

Reply:
This probably explains some of the debugging I'm doing for older SMs. Nice catch.

Reply:
Resolved with an amend.